tasks/src/feature-extraction/about.md

```diff
@@ -0,0 +1,34 @@
+## About the Task
+
+`Feature-Extraction` starts from initial set of measured data and builds derived values (features) intended to be informative and non-redundant, facilitating the subsequent learning and generalization steps in text, image & audio processing.
+The Hub contains over 2000 models for [feature-extraction](https://huggingface.co/models?pipeline_tag=feature-extraction&p=2&sort=downloads).
+
+## Use Cases
+It is used for downstream  tasks like 
+- Unsupervised Learning
+- Sentiment/Text classification
+- Image classification 
+- Audio Classification
+
+## Inference
+
+#### Feature Extraction
+
+```python
+## Loading the modules
+from transformers import pipeline
+#Fixing the chekpoint
+checkpoint = "facebook/bart-base"
+feature_extractor = pipeline("feature-extraction",framework="tf")
```

Comment:
```suggestion
feature_extractor = pipeline(model = "facebook/bart-base", task = "feature-extraction")
```

Reply:
I guess i am just using a custom model, so this should be fine?